### PR TITLE
Feature/orca 262 - Architecture Drawing and Documentation Updates

### PR DESCRIPTION
**Pull request by bhazuka**
## Summary of Changes

Update architecture drawings and documentation with new components and flow based on version 4 release. 

Addresses [ORCA-262: Update architecture drawings to include changes for reconciliation](https://bugs.earthdata.nasa.gov/browse/ORCA-262)

## Changes

* Updated the following files with new architecture information.
    * `website/docs/about/architecture/architecture-intro.mdx`
    * `website/docs/about/architecture/architecture-software-system.mdx`
    * `website/static/img/ORCA-Architecture-ORCA-System.svg`
    * `website/docs/about/architecture/architecture-archive-container.mdx`
    * `website/static/img/ORCA-Architecture-Archive-Container-Component.svg`
    * `website/docs/about/architecture/architecture-restore-container.mdx`
    * `website/static/img/ORCA-Architecture-Recovery-Container-Component.svg`
    * `website/docs/about/architecture/architecture-database-container.mdx`
    * `website/static/img/ORCA-Architecture-Database-Container-Component.svg`
* Added the following files
    * `website/docs/about/architecture/architecture-api-container.mdx` - New API container documentation
    * `website/static/img/ORCA-Architecture-API-Container-Component.svg` - Architecture drawing for API container
    * `website/static/files/ORCA-Architecture.drawio` - New file that allows users access to architecture through Draw IO
* Modified `website/sidebars.js` to add API Container page

## Type of change

- [X] This change is a documentation update

## How Has This Been Tested?

Please describe the tests (unit, integration, ad-hoc, or otherwise) that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

* Ran build of website through container to verify no issues with page builds.
* Visually inspected files and images served through the website locally.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
    - [X] Manual tests (outlined above)
- [X] My code has passed security scanning
    - [X] Snyk
    - [X] git-secrets

### CODE REVIEW

**Comment by Quarriers89** [comment]
Looks good to me, Brad.

File: website/docs/about/architecture/architecture-api-container.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The API Interface container contains all the components that allow an end user
+to access ORCA data including but not limited to recovery status information and
+ORCA catalog information. The architecture diagram below shows how generally, an
+operator goes through the Cumulus Dashboard to access the information. The dashboard
+makes API calls throuh an AWS API gateway which routes the call to the proper
```
Comment (rizbi-hassan) [inline website/docs/about/architecture/architecture-api-container.mdx:14]: check `throuh` spelling on line 14

**Review by AndrewDorn — APPROVED** [review]

**Review by rizbi-hassan — CHANGES_REQUESTED** [review]

**Comment by rizbi-hassan** [comment]
for ORCA-Architecture-Recovery-Container-Component.svg, the `request_files` and `copy_files_to_archive` block diagram description show `request-queue`. 
Shouldn't this be named `status-update-queue`?

File: website/docs/about/architecture/architecture-api-container.mdx (hunk 1 above)
Comment (bhazuka) [reply website/docs/about/architecture/architecture-api-container.mdx]: Fixed

**Review by rizbi-hassan — APPROVED** [review]

**Pull request merged**